Azure/sonic-buildimage
# DellEMC SFP Refactor

#### Why I did it 
Since SFP refactor has common API's, redundant platform API has to be removed.
#### How I did it
Removed redundant SFP API in DellEMC platforms.
#### How to verify it
Execute "sfpshow eeprom -d", verify syslogs pertaining to pmon, OIR tests etc..
[z9332f_ut.txt](https://github.com/Azure/sonic-buildimage/files/7459243/z9332f_ut.txt)


**Note:**
Please merge this PR only when sonic-platform-common submodule is updated with sonic_xcvr package in sonic-buildimage repo.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
